### PR TITLE
Fix branding styles

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

### What does this PR do?
Fix branding styles.

### What issues does this PR fix or reference?
fixes https://issues.redhat.com/browse/CRW-1693

### CODE REVIEW

File: assembly/branding/branding-che-defaults.css [hunk 1]
```diff
@@ -11,63 +11,49 @@
...
+@keyframes opacity {
+	0%,
+	100% {
+		opacity: 0;
+	}
+
+	50% {
+		opacity: 1;
+	}
+}
```
Comment (sleshchenko) [inline assembly/branding/branding-che-defaults.css:59]:
```suggestion
}

```

File: assembly/branding/branding-crw.css [hunk 2]
```diff
@@ -11,10 +11,10 @@
  */
 /*
 
-This file contains the branding changes needed for CodeReady Workspaces.
-It should override settings in default Che.
+This file contains the default branding in Eclipse Che Dashboard.
```
Comment (sleshchenko) [inline assembly/branding/branding-crw.css:14]:
```suggestion
This file contains the branding changes needed for CodeReady Workspaces Dashboard.
```

File: assembly/branding/branding-crw.css [hunk 3]
```diff
@@ -11,10 +11,10 @@
...
+See branding-crw.css for the overrides to be applied for CodeReady Workspaces.
```
Comment (sleshchenko) [inline assembly/branding/branding-crw.css:15]:
```suggestion
It should override settings in default Che.
```

File: assembly/branding/branding-crw.css [hunk 4]
```diff
@@ -24,50 +24,47 @@ See also https://github.com/redhat-developer/codeready-workspaces/blob/master/Je
...
+@keyframes opacity {
+	0%,
+	100% {
+		opacity: 0;
+	}
 
+	50% {
+		opacity: 1;
+	}
+}
```
Comment (sleshchenko) [inline assembly/branding/branding-crw.css:70]:
```suggestion
}

```

**Review by sleshchenko — APPROVED** [review]
I can't say for sure how it will look like in the end. Approving and we'll see the result on next CRW build.

File: assembly/branding/branding-crw.css (hunk 3 above)
Comment (olexii4) [reply assembly/branding/branding-crw.css:15]: fix description

**Pull request merged**